src/Stoa.ts

```diff
@@ -288,6 +289,7 @@ class Stoa extends WebService {
         this.app.get("/proposals/", isBlackList, this.getProposals.bind(this));
         this.app.get("/proposal/:proposal_id", isBlackList, this.getProposalById.bind(this));
         this.app.get("/validator/reward/:address", isBlackList, this.getValidatorReward.bind(this));
+        this.app.get("/convert-to-usd", isBlackList, this.convertToUSD.bind(this));
```

Comment:
Where is this endpoint used?

Reply:
this will be used for conversion of fee to $ amount.